.gitignore

```diff
@@ -91,3 +91,17 @@ unlinked_spec.ds
 !**/ios/**/default.pbxuser
 !**/ios/**/default.perspectivev3
 !/packages/flutter_tools/test/data/dart_dependencies_test/**/.packages
+
+# User-specific configurations
```

Comment:
We already ignore all of `.idea/` on line 17 of this file...  Am I missing something?

Reply:
OK. I didn't see it.